lib/main.dart

```diff
@@ -1,125 +1,145 @@
+              title: Text(
+                m.title,
+              ),
+              centerTitle: true,
+            ),
+            body: SingleChildScrollView(
+              child: SafeArea(
+                  top: true,
+                  bottom: true,
+                  left: true,
+                  right: true,
+                  child: Center(
+                    child: Column(
+                      children: [
+                        Stack(
+                          children: [
+                            Image.network(m.backdrop),
+                            Positioned(
+                              bottom: 0,
+                              right: 20,
+                              child: Text(
+                                m.originalTitle,
+                                style: const TextStyle(
+                                  fontWeight: FontWeight.bold,
+                                  fontSize: 25,
+                                  color: Colors.white,
+                                ),
+                                textAlign: TextAlign.center,
+                              ),
+                            ),
+                          ],
+                        ),
+                        Padding(
+                          padding: const EdgeInsets.all(16.0),
+                          child: Row(
+                            mainAxisAlignment: MainAxisAlignment.spaceEvenly,
+                            children: [
+                              MiddleText(s: 'Available on ${m.releaseDate}'),
+                              MiddleText(s: m.voteAverage),
+                            ],
+                          ),
+                        ),
+                        Row(
+                          mainAxisAlignment: MainAxisAlignment.spaceEvenly,
+                          children: m.getGenreElements(),
+                        ),
+                        const SizedBox(
+                          height: 20,
+                        ),
+                        Stack(
+                          children: [
+                            Image.network(m.poster),
+                            const Positioned(
+                              top: 15,
+                              right: 15,
+                              child: Column(
+                                children: [
+                                  Icon(
+                                    Icons.favorite_outlined,
+                                    size: 55,
+                                    color: Colors.red,
+                                  ),
+                                  FloatingActionButton(
+                                    onPressed: null,
+                                    child: Icon(Icons.add),
+                                  ),
+                                ],
+                              ),
+                            ),
+                            Positioned(
+                              width: 350,
```

Comment:
This width shouldn't be hard-coded, as it will depend on the Text content!

Reply:
Wrong comment above! You shouldn't specify this width based on your testing device's width. Wrap the `Positioned` with a `Padding` widget instead, and choose a suitable horizontal padding

Reply:
done